sound/soc/sof/intel/hda-stream.c

```diff
@@ -93,9 +94,6 @@ static int hda_setup_bdle(struct snd_sof_dev *sdev,
 		bdl++;
 		hstream->frags++;
 		offset += chunk;
-
-		dev_vdbg(sdev->dev, "bdl, frags:%d, chunk size:0x%x;\n",
-			 hstream->frags, chunk);
```

Comment:
any reason why this is removed without adding a tracepoint?

Reply:
@plbossart said it should be removed

Reply:
oh ok

Reply:
oh, ok x2